tsconfig.json

```diff
@@ -1,8 +1,13 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
   "exclude": ["packages"],
+  "include": ["src/**/*", "**/*.ts", "**/*.tsx"],
   "compilerOptions": {
     "module": "esnext",
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "typeRoots": ["./node_modules/@types", "./src/types"],
```

Comment:
Strange that you need this at all

Reply:
This must be a case of carelessly inserting AI-generated code since I didn't know that typeRoots is for custom types until I looked it up and I haven't (knowingly) created any custom types. 😬

I removed the line of code.